components/home/OurServices.vue

```diff
@@ -0,0 +1,182 @@
+            <v-image path="images/arcana-overview-auth.svg" alt="Arcana Auth" />
+          </div>
+          <div class="overview-feature">
+            <v-heading :level="2" class="overview-feature-title">
+              Arcana Auth
+            </v-heading>
+            <div class="overview-feature-description">
+              <v-text color="secondary" line-height="1.5" size="1.125rem">
+                Provide frictionless onboarding for web3 users using social auth
+                or passwordless logins. Arcana handles all aspects of key
+                management from key pair generation to secure non-custodial
+                handling through the arcana wallet so that your users don't have
+                to.
+              </v-text>
+            </div>
+          </div>
+        </v-stack>
+        <v-stack align="center" sm-align="start" gap="20vw" sm-gap="1rem">
+          <div class="overview-feature">
+            <div>
+              <v-heading :level="2" class="overview-feature-title">
+                Auth Store
+              </v-heading>
+            </div>
+            <div class="overview-feature-description">
+              <v-text color="secondary" line-height="1.5" size="1.125rem">
+                Store user data securely on Arcana's decentralised storage
+                network. Every asset uploaded to the arcana network is split up
+                and stored across our nodes so that there is no single point of
+                failure & your assets are protected against geo-specific
+                disasters and hacks.
+              </v-text>
+            </div>
+          </div>
+          <div class="icon-container">
+            <v-image
+              path="images/arcana-overview-store.svg"
+              alt="Arcana Storage"
+            />
+          </div>
+        </v-stack>
+        <v-stack align="center" sm-align="start" gap="20vw" sm-gap="1rem">
+          <div class="icon-container">
+            <v-image
+              path="images/arcana-overview-access.svg"
+              alt="Arcana Access"
+            />
+          </div>
+          <div class="overview-feature">
+            <div>
+              <v-heading :level="2" class="overview-feature-title">
+                Arcana Access
+              </v-heading>
+            </div>
+            <div class="overview-feature-description">
+              <v-text color="secondary" line-height="1.5" size="1.125rem">
+                Let your code or users decide who gets access to data. Arcana
+                Access uses DIDs for identification of data and to store,
+                on-chain, who is the owner of the data and who has access to it.
+              </v-text>
+            </div>
+          </div>
+        </v-stack>
+      </div>
+    </v-container>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'Features',
+  methods: {
+    openTechPage() {
+      window.open('/tech/', '_self')
+    },
```

Comment:
Is this used? Otherwise, let's remove it.

Reply:
Resolved at https://github.com/arcana-network/website/pull/100/commits/d522f0da68e3a55a6c4c5c2c387759f86164a1eb